test/grapple_test.exs

```diff
@@ -46,7 +54,7 @@ defmodule GrappleTest do
 
     test "sends a successful hook", %{hook: hook} do
       Hook.subscribe hook
-      [resp] = Hook.broadcast hook.topic
+      [%{response: resp}] = Hook.broadcast hook.topic
```

Comment:
Just realizing that this also justifies (at least) a minor version bump since we added new features. You could argue that we need a major version bump since we technically broke compatibility with older `Hook.broadcast` assignments. What do you think?